### PR TITLE
#1104 Move components and shared code to jac-kit

### PR DESCRIPTION
**Pull request by warrensearle**
Lots of changes here!

@tomlovesgithub please can you check this out and give it a thorough review.

### CODE REVIEW

**Review by tomlovesgithub — APPROVED** [review]
lot of changes as you say but looks fine locally here, made sure to test specific things in places where they are coming from the kit too
think its safe to merge 👍

File: src/filters.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { APPLICATION_STATUS, QUALIFYING_TEST } from '@/helpers/constants';
+import { APPLICATION_STATUS, QUALIFYING_TEST } from '@jac-uk/jac-kit/helpers/constants';
```
Comment (lloback) [inline src/filters.js:2]: didn't we move the filters???
We can separate this file into some reusable and some that are not and then import it here from the jac-kit.

And then, we can do the same on the apply side - import only the ones that are reusable.

**Review by lloback — APPROVED** [review]
@warrensearle I commented on some of the files, but it is good to go. Just read them as a side note...

File: src/filters.js (hunk 1 above)
Comment (warrensearle) [reply src/filters.js:2]: Agreed, am in the process of moving the filters and related tests

**Pull request merged**